R/mlr_graphs.R

```diff
@@ -0,0 +1,56 @@
+#' @title Dictionary of (sub-)graphs
+#'
+#' @usage NULL
+#' @format [`R6Class`] object inheriting from [`mlr3misc::Dictionary`].
+#'
+#' @description
+#' A simple [`Dictionary`][mlr3misc::Dictionary] storing objects of class [`Graph`].
+#' The dictionary contains a collection of often-used graph structures, and it's aim
+#' is solely to make often-used functions more accessible.
+#' Each `Graph` has an associated help page, which can be accessed via `?<key>`, i.e.
+#' `?bagging_pipeline`.
```

Comment:
For the other dictionary elements we usually have `?<dictname>_<key>`, e.g. `mlr_pipeops_colapply`

Reply:
Can do that